core/src/components/datetime/datetime-vars.scss

```diff
@@ -88,4 +92,12 @@
 
   // Prefix and Suffix
   --tds-datetime-ps-color: var(--tds-grey-600);
+
+  .tds-mode-variant-primary {
+    --tds-datetime-background: var(--tds-grey-900);
```

Comment:
Seems like the variable for background color is undefined when the option for mode variant is set to "Inherit from parent", making the input transparent. Is there a way we can introduce the variable outside of the primary/secondary scope?
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/08503176-3051-41ba-98d4-1aca9b87c022)

Reply:
Should be fixed now